base/number.jl

```diff
@@ -18,6 +18,7 @@ getindex(x::Number, I::Real...) = getindex(x, to_indexes(I...)...)
 unsafe_getindex(x::Real, i::Real) = x
 first(x::Number) = x
 last(x::Number) = x
+eachindex(x::Number) = 1
```

Comment:
this definition seems nicely meta: "the list of indices for iterating over a number can be found by iterating over the number 1"